nixos/modules/services/x11/desktop-managers/gnome3.nix

```diff
@@ -170,12 +170,16 @@ in {
     # If gnome3 is installed, build vim for gtk3 too.
     nixpkgs.config.vim.gui = "gtk3";
 
-    fonts.fonts = [
-      pkgs.dejavu_fonts pkgs.cantarell-fonts
-      pkgs.source-sans-pro
-      pkgs.source-code-pro # Default monospace font in 3.32
+    fonts.fonts = with pkgs; [
+      cantarell-fonts
+      dejavu_fonts
+      noto-fonts-emoji
+      source-code-pro # Default monospace font in 3.32
+      source-sans-pro
     ];
 
+    fonts.fontconfig.penultimate.enable = false;
```

Comment:
Does this interfere with emojis?

Reply:
It does, and it doesn't. I should probably mark this as wip since @jtojnar and I are still figuring this out. In particular people who try to use a different emoji font than the one we've configured will be stuck with this one. Thinking there should substitution rules, but I'm not too familiar with these pieces.

Reply:
@matthewbauer It appears penultimate is missing an emoji family.

Reply:
/cc @ttuegel

Reply:
> It appears penultimate is missing an emoji family.

I don't know what this means. :slightly_frowning_face:

Reply:
@ttuegel in 45-generic.conf & 60-generic.conf, there is `emoji` family defined.